src/JASSUB.cpp

```diff
@@ -354,6 +354,21 @@ class JASSUB {
     trackColorSpace = track->YCbCrMatrix;
   }
 
+  void newEmptyTrack() {
+    removeTrack();
+    track = ass_new_track(ass_library);
+    if (!track) {
+      fprintf(stderr, "JASSUB: Failed to start a track\n");
+      exit(4);
+    }
+
+    trackColorSpace = track->YCbCrMatrix;
+  }
+
+  void processLine(std::string line) {
+    ass_process_data(track, line.data(), line.length());
```

Comment:
I assume this allocates a new event as well?

Reply:
Yes